### ISSUE TITLE
Add support for bug trackers (Jira, Redmine, etc)

### ISSUE THREAD

**Issue by nzsmith1** [gauffininteractive/OneTrueError.Server#2]
It would be great to be able to push an incident type to a bug tracker system so that it can be managed in there.

**Comment by gauffininteractive** [comment]
It's fully possible to create a module for this.

1. Create a new class library for your favorite tracker
2. Subscribe on one of the [Incident events](https://github.com/gauffininteractive/OneTrueError.Server/tree/master/src/Server/OneTrueError.Api/Core/Incidents/Events)
3. Call your bug tracker

Example subscriber:

https://github.com/gauffininteractive/OneTrueError.Server/blob/master/src/Server/OneTrueError.App/Modules/ReportSpikes/CheckForReportPeak.cs

What would be nice though is to be able to configure the bug trackers during the installation wizard and the administration area. That do however need additional work to be able to add custom pages.

Same goes if a bug tracker link should be visible from the incident page.

**Comment by aussiealf** [comment]
Agreed this would be nice, I will have a look at the Incident Events, thanks for the point to an example subscriber. I am looking to integrate to Assembla so will see how I go and put in a Pull Request when I have something up and running.

**Comment by gauffininteractive** [comment]
There is a plugin api coming in 1.0

**Comment by aussiealf** [comment]
I see this is still open and is back in the OneTrueError days, I am now using coderr as our SaaS provider, will it be possible to have a ticket generation from error reports?

**Comment by jgauffin** [comment]
The plugin API is there but can't be used for Live. However, let me know which bug tracker you are using and I'll take a look at it.
